### DISCUSSION TITLE
Alternate LSP

### DISCUSSION THREAD

**Discussion by Moosems** [zed-industries/zed#17107] [Feature Requests]
I couldn't find any issues but there may be one out there I missed. Is there a way to use an alternate LSP and/or is that something planned?

**Comment by pjv** [comment]
see also #14731

**Comment by bajrangCoder** [comment]
From languages , language_server array you can disable/enable(or manage multiple ) LSP for any language (in settings)

Check [this](https://zed.dev/docs/configuring-languages#choosing-language-servers)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
